frontend/src/components/course/GuidedCourseView.js

```diff
@@ -35,12 +36,18 @@ const styles = theme => ({
     gridTemplateAreas:
       `"activeCourse contentHeader contentHeader"
        "activeCourse courses       courses"`,
-    gridTemplateColumns: '25% auto 25%',
+    // gridTemplateColumns: '25% auto 25%',
     gridTemplateRows: '64px calc(100vh - 58px - 64px)',
     '&.courseTrayOpen': {
       gridTemplateAreas:
         `"activeCourse contentHeader courseTray"
          "activeCourse courses       courseTray"`
+    },
+    '@media screen and (max-width: 1000px)': {
+      gridTemplateColumns: '32% auto 32%'
+    },
+    '@media screen and (min-width: 1000px)': {
+      gridTemplateColumns: '25% auto 25%'
```

Comment:
This media query should be removed and the default value above should be uncommented